### ISSUE TITLE
Vue removes aria-attributes that evaluate to `false`

### ISSUE THREAD

**Issue by obulat** [WordPress/openverse#2956]
## Description

The search bar in the header of the search page has a role of `combobox`. It has to have `aria-expanded="false"` in collapsed state. However, Vue removes the attribute if the values evaluates to `false`[^1]. 

To overcome this, we must convert the values for boolean aria attributes to strings.

Here are the places where search bar `aria-expanded` needs to be fixed:

https://github.com/WordPress/openverse/blob/2e9602812321458438582e46f61723e5659ad5c0/frontend/src/components/VHeader/VSearchBar/VSearchBar.vue#L27

 ```
:aria-expanded="`${isRecentVisible}`"
```

https://github.com/WordPress/openverse/blob/2e9602812321458438582e46f61723e5659ad5c0/frontend/src/components/VHeader/VHeaderMobile/VHeaderMobile.vue#L56
```
:aria-expanded="`${showRecentSearches}`"
```

There probably are other places, too, where `aria-expanded` should be fixed.

[^1]: [Bind boolean values to accessibility attributes](https://github.com/vuejs/vue/issues/7422)

## Reproduction
Run the lighthouse A11y check on the search page, and you will see this issue reported.

## Additional context

Mdn article states that the attribute is mandatory when the combobox is collapsed:

> Typically, the initial state of a combobox is collapsed, with aria-expanded="false" set. In the collapsed state, only the combobox element and, optionally a sibling button to invoke the popup, are visible. The [aria-expanded](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-expanded), with the value set to false, is required when collapsed, because it indicates to assistive technologies that the widget is expandable.

**Comment by dhruvkb** [comment]
This was apparently fixed in Vue 3. #411 can't come soon enough.

**Comment by obulat** [comment]
Closing this issue as the Nuxt 3 migration has fixed it.
However, it is necessary to audit the properties that should be removed instead of having `=false` value in other components.
